<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupsViewMain</class>
 <widget class="QMainWindow" name="GroupsViewMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Groups</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Please open a group from the Clipboard</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="groupContentView">
        <property name="iconSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="gridSize">
         <size>
          <width>220</width>
          <height>350</height>
         </size>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionAdd_Note">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
  </action>
  <action name="actionCreate_Group">
   <property name="text">
    <string>Create Group</string>
   </property>
  </action>
  <action name="actionAdd_to_Clipboard">
   <property name="text">
    <string>Add to Clipboard</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
